dotfile/Cargo.toml

```
@@ -0,0 +1,8 @@
+[package]
+name = "dotfile"
+edition = "2024"
+
+[dependencies]
+anyhow = "1.0.98"
```

```suggestion
anyhow = { workspace = true }
```
note: this was a different version than workspace! see?